prow/crier/reporters/github/reporter.go

```diff
@@ -150,10 +150,18 @@ func (c *Client) Report(_ *logrus.Entry, pj *v1.ProwJob) ([]*v1.ProwJob, *reconc
 
 	// TODO(krzyzacy): ditch ReportTemplate, and we can drop reference to config.Getter
 	err := report.Report(c.gc, c.config().Plank.ReportTemplateForRepo(pj.Spec.Refs), *pj, c.config().GitHubReporter.JobTypesToReport)
-	if err != nil && strings.Contains(err.Error(), "This SHA and context has reached the maximum number of statuses") {
-		// This is completely unrecoverable, so just swallow the error to make sure we wont retry, even when crier gets restarted.
-		err = nil
+	if err != nil {
+		if strings.Contains(err.Error(), "This SHA and context has reached the maximum number of statuses") {
+			// This is completely unrecoverable, so just swallow the error to make sure we wont retry, even when crier gets restarted.
+			log.WithError(err).Debug("Encountered unrecoverable error")
+			err = nil
+		} else if strings.Contains(err.Error(), "\"message\":\"Not Found\"") {
+			// "message":"Not Found" error occurs when someone force push, which is not a crier error
+			log.WithError(err).Debug("Cannot find endpoint")
```

Comment:
"Cannot find endpoint" doesn't seem like an accurate message. We wouldn't have received a 404 if that was the case, right? Something like "Could not find PR commit, skipping retries" would be more accurate, or just combine this with the previous conditional and use a generic debug message.

Reply:
done